README.md

```diff
@@ -57,15 +59,11 @@ module.exports = function () {
   // Register configuration
   registerConfig(ioc, path.join(__dirname, './config'));
 
-
   ioc.merge(spurCommon());
   ioc.merge(spurWeb());
 
   // register folders in your project to be auto-injected
-  ioc.registerFolders(__dirname, [
-    'controllers/',
-    'runtime/'
-  ]);
+  ioc.registerFolders(__dirname, ['controllers/', 'runtime/']);
```

Comment:
Not so much for this one, but in harness, can we please not rewrite the examples in markdown? This is not code so it's optimized for reading in the doc

Reply:
This was formatted by Prettier. `.md` files could be ignored but then you lose some of the benefits like table formatting, and consistency across `.md` files in the repo. 

I'm not convinced this is an issue. Depending the formatting setup of the target repo that you're copy/pasting into, it would get reformatted anyway right? i.e. If this was left unchanged and you pasted the example code into a repo using Prettier, it would be format like this.

Reply:
This is a public repo, it's for readability since we don't impose our formatting on anyone.